lib/mote_sms/transports/twilio_transport.rb

```diff
@@ -0,0 +1,69 @@
+require 'phony'
+require 'logger'
+
+module MoteSMS
```

Comment:
OT: somewhere you have to run `require 'twilio'` here, just adding it the gemfile is not enough